billing/src/main/java/com/appcoins/wallet/billing/repository/entity/TransactionData.kt

```diff
@@ -28,6 +28,7 @@ class TransactionData @JvmOverloads constructor(@SerializedName("type")
   enum class TransactionType {
     DONATION,
     INAPP,
+    SUBS,
```

Comment:
Consider adding a comment here for the time when the subscriptions are supported by the backend and make sure that this will be a TransactionType